pkgs/top-level/all-packages.nix

```
@@ -5234,6 +5234,8 @@ in
 
   pydb = callPackage ../development/tools/pydb { };
 
+  pydf = callPackage ../development/misc/pydf { };
```

From the ofborg logs  in the Checks tab, this should be:

```suggestion
  pydf = callPackage ../applications/misc/pydf { };
```

This PR is looking pretty good!

---

silly mistake due to copy/paste, fixed